.github/workflows/subgraph-documentation-update.yml

```diff
@@ -42,10 +42,10 @@ jobs:
           path: developer-portal
           token: ${{ secrets.ARABOT_PAT }}
       - name: Remove the docs/subgraph folder in the developer-portal
-        run: rm -rf $GITHUB_WORKSPACE/developer-portal/docs/subgraph
+        run: rm -rf $GITHUB_WORKSPACE/developer-portal/versioned_docs/version-1.3.0/osx/subgraph
       - name: Copy the docs/developer-portal folder to the developer-portal
-        run: cp -R packages/subgraph/docs/developer-portal $GITHUB_WORKSPACE/developer-portal/docs/subgraph
-      - name: Copy the generated schema-introspection.json to the developer-portal
+        run: cp -R packages/subgraph/docs/developer-portal $GITHUB_WORKSPACE/developer-portal/versioned_docs/version-1.3.0/osx/subgraph
```

Comment:
Shouldn't this be 1.4.0?

Reply:
For the moment we agreed to have both SDK and Subgraph living inside OSx's 1.3.0. 
Moreover with the refactoring 1.4.0 will be the new discussed format and we didn't talk about including Subgraph.